scripts/generate_learning_path_markdown.js

```diff
@@ -60,8 +92,9 @@ const getYouTubeCode = (section, articleNumber) => {
       writeMarkdownFile(indexWritePath, indexFrontMatter, indexContent)
 
       articles.forEach(async (article) => {
-        const articleTitle = article.asciiDoc.match(/== (.*)/)[1]
+        const articleTitle = getArticleTitle(article)
         const articleNumber = basename(article.filePath).split('-')[0]
+        const articleText = getArticleText(article)
```

Comment:
What's the overall strategy here on article text vs. title and when to use each (and why to change, here)?